natlas-server/.scss-lint.yml

```diff
@@ -0,0 +1,260 @@
+# Default application configuration that all configurations inherit from.
+
+scss_files: "**/*.scss"
+exclude: 'node_modules/**'
+plugin_directories: ['.scss-linters']
+
+# List of gem names to load custom linters from (make sure they are already
+# installed)
+plugin_gems: []
+
+# Default severity of all linters.
+severity: warning
```

Comment:
Is this the just give up mode?

Reply:
No idea, it's the default config with a few minor modifications 😄